src/components/WrapForm/index.tsx

```diff
@@ -1,4 +1,4 @@
-import { FC, FormEvent, MouseEvent, useEffect, useState } from 'react'
+import { FC, FormEvent, MouseEvent, useEffect, useRef, useState } from 'react'
```

Comment:
TODO in the future: restructure some components to fix HMR issues:
```
1:45:55 PM [vite] hmr invalidate /src/providers/Web3Provider.tsx Could not Fast Refresh. Learn more at https://github.com/vitejs/vite-plugin-react-swc#consistent-components-exports
1:45:55 PM [vite] hmr update /src/App.tsx, /src/hooks/useWeb3.tsx
1:45:55 PM [vite] hmr invalidate /src/hooks/useWeb3.tsx Could not Fast Refresh. Learn more at https://github.com/vitejs/vite-plugin-react-swc#consistent-components-exports
1:45:55 PM [vite] hmr update /src/components/Layout/index.tsx, /src/pages/ConnectWallet/index.tsx, /src/providers/WrapFormProvider.tsx, /src/pages/Wrapper/index.tsx, /src/pages/Transaction/index.tsx, /src/components/Account/index.tsx
1:45:55 PM [vite] hmr invalidate /src/providers/WrapFormProvider.tsx Could not Fast Refresh. Learn more at https://github.com/vitejs/vite-plugin-react-swc#consistent-components-exports
1:45:55 PM [vite] hmr update /src/App.tsx, /src/hooks/useWrapForm.tsx
1:45:55 PM [vite] hmr invalidate /src/hooks/useWrapForm.tsx Could not Fast Refresh. Learn more at https://github.com/vitejs/vite-plugin-react-swc#consistent-components-exports
```

Reply:
Fixed, extension were wrong in hooks folder(`tsx` instead of `ts`). Thats why `vite-plugin-react-swc` was complaining.